index.js

```diff
@@ -214,8 +214,8 @@ const styles = StyleSheet.create({
     borderColor: '#C8C7CC',
   },
   valueText: {
-    height: (Platform.OS == 'ios' ? 20 : 35),
-    fontSize: (Platform.OS == 'ios' ? 16 : 14),
+    height: (Platform.OS == 'ios' ? 20 : 60),
```

Comment:
seems Android uses different heights than iOS on TextInputs:

[height20](https://s15.postimg.org/40756q9iz/Screen_Shot_2017_03_22_at_12_04_09.png)
[height60](https://s15.postimg.org/hslk2d0aj/Screen_Shot_2017_03_22_at_12_03_38.png)

There is no problem if you use a real big value (f.e. 2000), the field is still rendered fine.

I've changed from 35 to 60 to avoid problems with bigger fontSize (14 to 16)
Now the font size is the same on both platforms.